.gitignore

```diff
@@ -1,2 +1,1089 @@
 .idea/
 .vscode/
+### Momentics template
```

Comment:
Please add only necessary entries

Reply:
I will update it with just windows, Linux, macOS filesystems and also go output files like .exe, .out or .test, and I will add vim, emacs, and sublime too to had more useful .gitignore file

Reply:
Please do that in a _separate_ PR as this changes have nothing to do with Go file name conventions.

Reply:
@cclauss, you're right. I restore it as default. I'll make another pull request for it later.